docs/reading.md

```diff
@@ -10,6 +10,16 @@ pageClass: routes
 
 <Route author="HenryQW" example="/allpoetry/newest" path="/allpoetry/:order?" :paramsDesc="['排序方式, `best` 或 `newest`, 缺省 `best`']"/>
 
+## Free Computer Books
+
+### 精选上新
+
+<Route author="cubroe" example="/freecomputerbooks" path="/freecomputerbooks" radar="1" />
+
+### 当前书单
+
+<Route author="cubroe" example="/freecomputerbooks/compscAlgorithmBooks" path="/freecomputerbooks/:category" :paramsDesc="['分类标识符，为源站URL路径中的HTML文件名（去除`.html`后缀）']" radar="1" />
```

Comment:
```suggestion
<Route author="cubroe" example="/freecomputerbooks/compscAlgorithmBooks" path="/freecomputerbooks/:category?" :paramsDesc="['分类标识符，为源站URL路径中的HTML文件名（去除`.html`后缀）']" radar="1" />
```
https://github.com/DIYgod/RSSHub/blob/6cd5be1d81851ee390605f8db3426d2eedc762d5/lib/v2/freecomputerbooks/maintainer.js#L2

Reply:
@TonyRL 

The optionality actually is only visible on the underlying implementation level; on the UI (as interface) level, it differs from the root path `/freecomputerbooks` above with a different section title.

Let me know if you do need the uniformity between `router.js` and the document, or otherwise the overlapping between the two document sections would be confusing for careful readers. Or, perhaps you prefer my explicitly splitting the router entry into the two of
- `/`
- `/:category`

...?

Reply:
You can update the `maintainer.js` into two entries
'/':['cubroe']
'/:category':['cubroe']

Reply:
Squashed all suggestions, re-committed and pushed. Please review again. Thanks!